test/evergreen.yml

```diff
@@ -28,6 +29,39 @@ exec_timeout_secs: 21600 # 6 hrs
 #######################################
 
 functions:
+  "setup environment":
+    - command: expansions.update
```

Comment:
Thanks for introducing this Evergreen command to me. While it's not super intuitive for me to understand at which stage these expansion updates are applied, and the need of calling the update key (e.g. PREPARE_TEST_ENV) explicitly in the task definition, it could be a similar experience for other readers. A comment could be helpful.

Reply:
done!